doc/user/bgp.rst

```diff
@@ -2841,6 +2841,23 @@ L3VPN SRv6
    Specify the SRv6 locator to be used for SRv6 L3VPN. The Locator name must
    be set in zebra, but user can set it in any order.
 
+General configuration
+^^^^^^^^^^^^^^^^^^^^^
+
+Configuration of the SRv6 SID used to advertise a L3VPN for both IPv4 and IPv6
+is accomplished via the following command in the context of a VRF:
+
+.. clicmd:: sid vpn per-vrf export (1..255)|auto
```

Comment:
is this truly only for export?

Reply:
Yes, it is only for export.

``sid vpn per-vrf export (1..255)|auto`` only enables exporting the routes from a VRF to the VPN RIB using a given SRv6 SID. This command has no impact on the imported routes. In order to enable and configure the import of routes from the VPN RIB to a VRF, you need to use the ``import vpn`` and ``rt vpn import`` commands already supported in bgpd (in the context of an address-family). 
Note: this command already exists before. It was used to do export per-address family. We reuse the same syntax to do export per-vrf.